menuflow/utils/util.py

```diff
@@ -47,6 +49,50 @@ def is_room_id(cls, room_id: RoomID) -> bool:
         """
         return False if not room_id else bool(match(f"^!{cls._main_matrix_regex}+$", room_id))
 
+    @classmethod
+    async def get_tasks_by_name(self, task_name: str) -> Task:
+        """It returns a task object from the current event loop, given the task's name
+
+        Parameters
+        ----------
+        task_name
+            The name of the task to find.
+
+        Returns
+        -------
+            An specific task.
+
+        """
+
+        tasks = asyncio.all_tasks()
+        for task in tasks:
+            if task.get_name() == task_name:
+                return task
+
+    @classmethod
+    def is_within_range(self, number: int, start: int, end: int) -> bool:
+        """ "Return True if number is within the range of start and end, inclusive."
+
+        Parameters
+        ----------
+        number : int
+            the number to check
+        start : int
+            The start of the range.
+        end : int
+            The end of the range.
+
+        Returns
+        -------
+            A boolean value
+
+        """
+
+        if not (number or start or end):
```

Comment:
You should change this validation because the wanted behaviour is: if you send some empty variable you must return False.